### ISSUE TITLE
UI: indicate end of a pipeline-step in output

### ISSUE THREAD

**Issue by flosell** [flosell/lambdacd#4]
If users are following the output of a particular build-step, the pipeline overview quickly gets out of view, leaving the user without an indication of how the build step they are watching is doing.

If the build-step itself doesn't give a good indication that it's finished, users might wait for updates to the output, not noticing the build step is already done. 

Add a message like `BUILD STEP FINISHED` to the end of the output display once the build step is done.
